src/NuGet.Core/NuGet.ProjectModel/ProjectRestoreMetadataFrameworkInfo.cs

```diff
@@ -44,7 +44,7 @@ public override int GetHashCode()
 
             hashCode.AddObject(FrameworkName);
             hashCode.AddStringIgnoreCase(TargetAlias);
-            hashCode.AddSequence(ProjectReferences);
+            hashCode.AddUnorderedSequence(ProjectReferences);
```

Comment:
How does this change work with IDE0051? I don't see any changes to HashCodeCombiner here.